Shades-ColorGenerator/README.md

```diff
@@ -0,0 +1,5 @@
+# Shades-ColorGenerator-
+
+## A website with functionality to generate and copy the different shades of colors.
+
+![](src/shades.PNG)
```

Comment:
- Add a snapshot of your project in  gif,jpg, png 
- write step set up the project 
-  Write Author name in last

Reply:
I guess it should look good.